wrappers/c/zxing.cpp

```diff
@@ -0,0 +1,155 @@
+#include "zxing.h"
+
+#include <ZXing/BarcodeFormat.h>
+#include <ZXing/DecodeHints.h>
+#include <ZXing/ImageView.h>
+#include <ZXing/ReadBarcode.h>
+#include <ZXing/Result.h>
+
+extern "C"
```

Comment:
I updated the .clang-format file to not indent `extern "C"` blocks. Please reapply.